src/components/ui/badge/index.tsx

```diff
@@ -67,4 +67,4 @@ class BadgeComponent extends Component<IProps> {
   }
 }
 
-export const Badge = injectStyle(styles, { injectTheme: true })(BadgeComponent);
+export default injectStyle(styles, { injectTheme: true })(BadgeComponent);
```

Comment:
(@cino - I know this is still WIP, but) Could this pattern of changes be done outside of this PR (as a separate PR) - so that this one becomes lighter?

Reply:
Yeah I agree, will look into creating a separate PR for these later today and keep in mind for other changes.

Reply:
There was a different PR that was created and merged. Maybe this PR branch needs to be rebased?